SetupDataPkg/SetupDataPkg.dec

```diff
@@ -38,5 +40,7 @@
 [PcdsFixedAtBuild]
 
 [PcdsDynamic, PcdsDynamicEx]
+  # Profile driver will receive the active profile GUID and populate this PCD
```

Comment:
Probably want to add a comment on platform is recommended populate a default value.

Reply:
Added.